src/v/utils/base64.cc

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#include "utils/base64.h"
+
+#include "vassert.h"
+
+#include <seastar/core/sstring.hh>
+
+#include <libbase64.h>
+
+// Required length is ceil(4n/3) rounded up to 4 bytes
+static inline size_t encode_capacity(size_t input_size) {
+    return (((4 * input_size) / 3) + 3) & ~0x3U;
+}
+
+bytes base64_to_bytes(std::string_view input) {
+    bytes output(bytes::initialized_later{}, input.size());
```

Comment:
Perhaps `decode_capacity` as the inverse of `encode_capacity`, would reduce allocation size here.